analyses/bsve/search/minerva.json

```diff
@@ -0,0 +1 @@
+{ "minerva": { "analysis_type": "bsve_search" } }
```

Comment:
What I was worried about is creating a Romanesco analysis where there isn't one.  e.g., For the bsve search, which is currently a local job, would it be a problem to call romanesco.load?  I pointed out in another comment where we are calling load.

Even if it isn't a problem to call romanesco.load, it is a bit misleading to call that on an analysis that shouldn't be run under romanesco.

I agree generally and would like to remove the extra minerva.json if possible.